.circleci/config.yml

```diff
@@ -359,6 +359,26 @@ jobs:
       - store_artifacts:
           path: /go/src/github.com/tendermint/tendermint/tendermint-*.tar.gz
 
+  # Get Node.js and test REST implementation against the swagger documentation of the SDK version used
+  contract_tests:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - restore_cache:
+          key: v4-pkg-cache
+      - restore_cache:
+          key: v3-tree-{{ .Environment.CIRCLE_SHA1 }}
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Contract tests
```

Comment:
Can we be more specific about what sort of contract?

Reply:
```suggestion
          name: Test Swagger
```

Reply:
It's basically a way to check the implementation against a document 'contract' instead of an integration test, usually more expensive
In theory, the documentation will describe the specs we want to follow, and we test the implementation against the spec.